internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -1,7 +1,7 @@
 {{ $username := fact "username" }}
 {{ $password := fact "password" }}
 {{ $apm_enabled := fact "apm_enabled" }}
-version: '2.3'
+version: '2.4'
```

Comment:
Does the version also need to be changed in the serverless docker compose file?

Reply:
Right, It needs to be updated
Thanks, I'll change it!